views/login.ejs

```diff
@@ -33,8 +33,8 @@
 				<input id='passwordInput' type="password" class="form-control" name="password" value="" required/>
 			</div>
 		</div>
+		<a href="/" class="btn btn-primary">Back</a>
```

Comment:
Really stupid complaint but when I type in details I am used to entering the email, pressing tab, entering the password, pressing tab, then pressing return. Maybe Butterfield could get annoyed if he does the same and ends up back at the main screen. Maybe have back button after/beneath submit? Other than that I'm pretty sure all buttons work